### PR TITLE
Aimenu

### PR DESCRIPTION
**Pull request by niko227**
unified main menu with A.I. #255
Done:
prefabs de MediumAI y HardAI creados.
leer la dificultad desde el menú y instanciar el prefab correspondiente

Para que funcione bien hay que cargar los prefabs de las IA en camera.

### CODE REVIEW

File: Assets/Scripts/Controller/Loader.cs [hunk 1]
```diff
@@ -7,13 +7,29 @@
...
+			//Instantiate GameController prefab
+			Instantiate (gameController);
+
+	}
+
+	private void Start(){
+		if (PersistentValues.difficulty == 1) {
+			Instantiate (EasyAI);
+		} else if (PersistentValues.difficulty == 2) {
+			Instantiate (EasyAI);
```
Comment (Solero93) [inline Assets/Scripts/Controller/Loader.cs]: Shouldn't this be

> Instantiate (MediumAI);

File: Assets/Scripts/Menu/SubMenu.cs [hunk 2]
```diff
@@ -40,6 +40,7 @@ public class SubMenu : MonoBehaviour {
 	public void OnMedium(){
 		PersistentValues.difficulty = 2;
 		StartCoroutine(playAndLoad(5));
+
```
Comment (Solero93) [inline Assets/Scripts/Menu/SubMenu.cs]: Delete this extra space, it could lead to unnecessary conflicts

**Review by Solero93 — CHANGES_REQUESTED** [review]
Another thing... how have you not changed the scene to include the Instantiation of AI's? I mean... you have reassigned the attributes of Loader inside the camera. Make sure you've saved the scene, because I think you haven't

File: Assets/Scripts/Controller/Loader.cs (hunk 1 above)
Comment (niko227) [reply Assets/Scripts/Controller/Loader.cs]: cierto...

**Review by Solero93 — APPROVED** [review]
Looks good to me! Well done!

**Pull request merged**